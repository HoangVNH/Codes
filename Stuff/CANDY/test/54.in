10
100
XXDXVVVXDXDVXVDVDXVDVDVXDDDDVXDXXDXVDXXXXXXVDDVXVDXVXXDDDVXVVVDXVXVVVVVVVDXDDXVVVVVXDDXVDDDVVXDVDVVV
59
XDVXDVDVXVXXDVDDVDDVXDDDXXXDDXVDVDVVVVDDVVVDXXVXDVXVDDVVVXX
86
DDXDVVDVVDVVXXDXXDVDXVXDXXVDDXVDXDVXXVDXXXVXVDVVVDXXDXXVXVDXDDDDDVXXXVDXXDVXDXXXVDDXVV
58
XVXVDXVVVXXVXXXVDVXXDXVXDVDDDVVXXVXVVVDXXVDVXVVVDVDDVVXDDV
98
VXDDDVVDDVXVDVDVXDXXVDXVVVDVXXDXDXVVDVDVVDXDVXVDXXDDDXDVXDDDDDVXDXDDDDDDXXDXXDXVVXXDXVDVDVVDVDDXVD
74
VVDDVDVDXXDXDXVDDVXDVVXDVDVVVDDVXVDXDVDXVVDXVVDVVVVXVXVDVVXVVDVVVDVDDDVDDV
89
VVXXDVVVXVXXVXDXDDVXXDDXVDVVXDDXXVVDDVDXVVVXDVXXVXVDVVVVDXXDVDDVVDDXDVVVXDXDDDXDDVXVDDVDD
81
DDXDXDDDXXVDDVDXXDDVXDVXDVVVXXXDDXDXVVVXDVVXVDDVXDXVDDXDVXXVDDVXXXXXVDVXXXDDXVDVV
84
VXVVVXXXDXVDDVDVDDXVVDVVDDXXXVDDDXVVXDDDDDDXXVDDVVDVXDVXVDVDVVDDVXXDVVDVDDXVVXXDVVXX
83
XXXXXVXVDDXDVVVVXXXXXDVDXDXVDDVDDXVDXXXXVXVVDDVVXXVDVXDXVXDXXXXXXDDXDXVDVDXDVXVVVVX
