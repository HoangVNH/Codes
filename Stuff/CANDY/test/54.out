99
58
85
57
97
73
87
79
83
82
